test/util/manifests/api.go

```diff
@@ -0,0 +1,68 @@
+package manifests
```

Comment:
I don't believe this file is necessary in the scope of this PR.

Reply:
Need to think about this. Agree, this need to change